README.md

```diff
@@ -326,6 +326,56 @@ Translations of the guide are available in the following languages:
   end
   ```
 
+<a name="multi-condition-case-when"></a>
+* put multiple when conditions on separate lines
+particularly where the conditions form long, complicated lines
+<sup>[[link](#multi-condition-case-when)]</sup>
+
+  ```Ruby
+  # good
+
+  case token
+  when :star_op
```

Comment:
Well, the commit diff affect other lines than where the comments are, so these remain.  But, updated with 2186b71